devito/types/grid.py

```diff
@@ -515,13 +603,12 @@ def __subdomain_finalize__(self, grid, **kwargs):
             else:
                 try:
                     # Case ('middle', int, int)
-                    side, thickness_left, thickness_right = v
+                    side, ltkn, rtkn = v
```

Comment:
Why!?

Reply:
Maybe this is something I just have to get used to, but it completely threw me when I saw it elsewhere